include/cs_threads.h

```diff
@@ -0,0 +1,239 @@
+/*
```

Comment:
This and other headers seems to have PUBLIC functions that aren't getting included into Doxygen.  We need to audit and make sure the generated doxygen docs include everything we're making PUBLIC.

Reply:
updated doxyfile